ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -3131,7 +3133,7 @@
 				ENABLE_BITCODE = YES;
 				ENABLE_TESTABILITY = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Why was the macOS deployment target changed?

That should probably be a separate PR/discussion.